tradeexecutor/visual/utils.py

```diff
@@ -144,12 +144,14 @@ def export_trades_as_dataframe(
 
         # Crop
         if start or end:
-            if not t.started_at:
+            s = t.started_at or t.opened_at
```

Comment:
Can you change so that `t.opened_at` is preferred variable as it should always match the strategy cycle timestamp?